lib/TraackrApi/TraackrApiObject.php

```diff
@@ -2,256 +2,299 @@
 
```

Comment:
Set the diff settings to hide whitespace changes. Will be much easier to go through that way.

Reply:
thanks for the tip, made the difference